hack/build/docker/builder/Dockerfile

```diff
@@ -52,6 +52,16 @@ COPY output-bazel-arch.sh /output-bazel-arch.sh
 
 RUN curl -L -o /usr/bin/bazel https://github.com/bazelbuild/bazel/releases/download/${BAZEL_VERSION}/bazel-${BAZEL_VERSION}-linux-$(sh /output-bazel-arch.sh) && chmod u+x /usr/bin/bazel
 
+RUN if test "${ARCH}" = amd64; then \
```

Comment:
hmm, `amd64`?

Reply:
There is no
```
ARG ARCH
```
declaration in the Dockerfile, so this should be an unset variable.

Additionally you would need to declare it as ``--build-arg ARCH=<architecture>`.

I think easier in this case would be to simply use the [pre-defined build variable](https://docs.docker.com/build/building/variables/#pre-defined-build-arguments) `TARGETARCH`